libs/core/datastructures/CMakeLists.txt

```diff
@@ -34,15 +34,17 @@ set(datastructures_headers
     hpx/datastructures/variant_helper.hpp
 )
 
+# cmake-format: off
 set(datastructures_compat_headers
-    hpx/datastructures.hpp
-    hpx/traits/supports_streaming_with_any.hpp
-    hpx/traits/is_tuple_like.hpp
-    hpx/util/any.hpp
-    hpx/util/optional.hpp
-    hpx/util/tagged.hpp
-    hpx/util/tuple.hpp
+    hpx/datastructures.hpp => hpx/modules/util.hpp
```

Comment:
I think this one was most probably a mistake... as in it should point to `hpx/modules/datastructures.hpp` instead.